README.md

```diff
@@ -4,22 +4,28 @@
 
 The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from a bash shell terminal:
 
-### On macOS and Linux
+### System Requirements
```

Comment:
It would be good to briefly mention the option of running the app via Vagrant

Reply:
updated